design-system/scripts/generate-design-tokens.js

```diff
@@ -79,6 +80,7 @@ Object.entries(definitions.decisionGroupsByTheme).forEach(
         if (match) {
           const componentGroup = match[2];
           const state = match[3];
+          const variant = match[4];
```

Comment:
In theory this should be one.

```suggestion
          const stateOrVariant = match[3];
```